modules/shared/config/config.go

```diff
@@ -289,7 +295,14 @@ func DefaultConfig() *Config {
 	cfg.Dashboard.Session.Cookie.SameSite = http.SameSiteLaxMode
 	cfg.Dashboard.UI.ClusterAPIEnabled = true
 
-	cfg.ClusterAPI.Addr = ":8080"
+	cfg.ClusterAPI.HTTP.Enabled = true
+	cfg.ClusterAPI.HTTP.Address = ""
+	cfg.ClusterAPI.HTTP.Port = 8080
+	cfg.ClusterAPI.HTTPS.Enabled = true
```

Comment:
We should disable https by default otherwise the server will throw errors on startup due to the empty certfile and keyfile.